### PR TITLE
Added Custom Redirect create and update and File read

### PR DESCRIPTION
**Pull request by hilltro**

### CODE REVIEW

File: docs/kb/api-version-3/custom-redirects.md [hunk 1]
```diff
@@ -75,6 +75,8 @@ For a complete list of fields involved in user operations, see the [Custom Redir
 | **Operation** | **URL Format**                             | **Required Parameters** | **Description**  |
 | ------------- | ------------------------------------------ | ----------------------- | -----------------|
 | `read` | `/api/customRedirect/version/3/do/read/id/`**_`<id>`_**`?...` | `user_key, api_key, id` | Returns the data for the custom redirect specified by `<id>`. `<id>` is the Pardot ID of the target custom redirect. |
+| `create` | `/api/customRedirect/version/3/do/create?... | `user_key, api_key, id` | Creates a new custom redirect using the specified data. |
```
Comment (hdartz) [inline docs/kb/api-version-3/custom-redirects.md]: missing an apostrophe after: `/api/customRedirect/version/3/do/create?...

Comment (hilltro) [reply docs/kb/api-version-3/custom-redirects.md]: thank you

File: docs/kb/api-version-4/custom-redirects.md [hunk 2]
```diff
@@ -75,6 +75,8 @@ For a complete list of fields involved in user operations, see the [Custom Redir
 | **Operation** | **URL Format**                             | **Required Parameters** | **Description**  |
 | ------------- | ------------------------------------------ | ----------------------- | -----------------|
 | `read` | `/api/customRedirect/version/4/do/read/id/`**_`<id>`_**`?...` | `user_key, api_key, id` | Returns the data for the custom redirect specified by `<id>`. `<id>` is the Pardot ID of the target custom redirect. |
+| `create` | `/api/customRedirect/version/4/do/create?... | `user_key, api_key, id` | Creates a new custom redirect using the specified data. |
```
Comment (hdartz) [inline docs/kb/api-version-4/custom-redirects.md]: same missing apostrophe here

**Review by hdartz — CHANGES_REQUESTED** [review]
one more missing apostrophe but outside of that, looks good.

**Review by hdartz — CHANGES_REQUESTED** [review]
Heads up Troy - looks like you fixed version/3 but you're still missing an apostrophe for version/4/do/create

**Comment by ovidio-peduri** [comment]
@hdartz

File: docs/kb/api-version-4/files.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+| ------------- | ------------------------------------------ | ----------------------- | -----------------|
+| `read` | `/api/file/version/4/do/read/id/`**_`<id>`_**`?...` | `user_key, api_key, id` | Returns the data for the file specified by `<id>`. `<id>` is the Pardot ID of the target file. |
+
+
+## XML Response Format
+
+```
+<rsp stat="ok" version="1.0">
+    <file>
+      <id>43</id>
```
Comment (Razenghan) [inline docs/kb/api-version-4/files.md]: Does `salesforce_fid` need to be reflected here and/or v4?

**Review by Razenghan — CHANGES_REQUESTED** [review]

File: docs/kb/api-version-3/custom-redirects.md [hunk 4]
```diff
@@ -75,6 +75,8 @@ For a complete list of fields involved in user operations, see the [Custom Redir
 | **Operation** | **URL Format**                             | **Required Parameters** | **Description**  |
 | ------------- | ------------------------------------------ | ----------------------- | -----------------|
 | `read` | `/api/customRedirect/version/3/do/read/id/`**_`<id>`_**`?...` | `user_key, api_key, id` | Returns the data for the custom redirect specified by `<id>`. `<id>` is the Pardot ID of the target custom redirect. |
+| `create` | `/api/customRedirect/version/3/do/create?...` | `user_key, api_key, id` | Creates a new custom redirect using the specified data. |
```
[the remark this answers is not in the document]
Comment (hilltro) [reply docs/kb/api-version-3/custom-redirects.md]: Good catch.

**Pull request closed without merging**